english/fantasy/book.md

```diff
@@ -0,0 +1,7 @@
+“ I am not done yet!”
+
+The mangled body twitched, trying to move.  Trying to stand.  Positronic units cut power to severed actuators, rerouting dwindling power to functioning systems.  The charred body moved slowly, broken and twisted, it moved by will power alone.  Something snapped.  Its shielded ocular units flickered as systems died.  A final burst of power surged through damaged pathways.  “ I am not done…”  The unit’s power cells, drained completely kicked to reserve power.  Enough to shutdown primary systems.  Raw connections stopped sparking.  The air around the body smelled of ozone.  A figure in black rose from a crouching position, check the vitals readout on the fallen target on its HUD, shutdown its pulse cannon and rerouted power to its transtemporal units.  The black head turned surveying the surrounding area sifting through sensory data that it was receiving and then blinked out of existence.  Slowly, woodland animals crept from their shelters.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.